.gitignore

```
@@ -23,3 +23,16 @@ go.work.sum
 
 # Text file with secrets and links
 secrets.txt
+
+# data and des folders and files
+controlledData/
+controlledDes/
+encodedData/
+encodedDes/
+gridData/
+gridDes/
+savedImages/
+sensorData/
+sensorDes/
+wheeledData/
+wheeledDes/
```

realized that making the folder itself part of the gitignore would meant that peeps using this repo would have to create the folders themselves.

that might be okay, but if its not we can do this

```suggestion
controlledData/*
controlledDes/*
encodedData/*
encodedDes/*
gridData/*
gridDes/*
savedImages/*
sensorData/*
sensorDes/*
wheeledData/*
wheeledDes/*
```

---

going to have to jump through some hoops to make this work because it doesn't seem to recognize the folder as a new change for GitHub unless there's a file in it, but this ignores the file so therefore ignores the folder